php/php-codesniffer-standard/VIISON/Sniffs/Strings/DateMethodsInSqlSniff.php

```diff
@@ -0,0 +1,52 @@
+<?php
+namespace VIISON\StyleGuide\PHPCS\Standards\VIISON\Sniffs\Strings;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+
+// phpcs:disable VIISON.Strings.DateMethodsInSql.NoNow
+// phpcs:disable VIISON.Strings.DateMethodsInSql.NoCurrentTimestamp
+// phpcs:disable VIISON.Strings.DateMethodsInSql.UtcTimestampOnlyWithArgument
+
+/**
+ * This sniff disallows the usage of the method NOW() and CURRENT_TIMESTAMP() in SQL queries as they have weird
+ * timezone behavior. It suggests using UTC_TIMESTAMP() instead.
+ */
+class DateMethodsInSqlSniff implements Sniff
+{
+    /**
+     * @inheritdoc
+     */
+    public function register()
+    {
+        return [
+            T_CONSTANT_ENCAPSED_STRING,
+            T_HEREDOC,
+            T_NOWDOC,
+        ];
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $token = $phpcsFile->getTokens()[$stackPtr];
+        $content = $token['content'];
+
+        if (mb_stripos($content, 'NOW(') !== false) {
+            $error = 'The usage of the method NOW() in SQL queries is not allowed. Use UTC_TIMESTAMP() instead.';
```

Comment:
Since this same sniff enforce the use of millisecond precision with `UTC_TIMESTAMP()` we should mention this in the error message.
```suggestion
            $error = 'The usage of the method NOW() in SQL queries is not allowed. Use UTC_TIMESTAMP(3) instead.';
```